src/components/discover-sheet/ListsSection.js

```diff
@@ -115,9 +116,11 @@ export default function ListSection() {
 
   const handleSwitchList = useCallback(
     (id, index) => {
-      LayoutAnimation.configureNext(
-        LayoutAnimation.create(200, 'easeInEaseOut', 'opacity')
-      );
+      if (IS_TESTING !== 'false') {
```

Comment:
do you want this if true?

Reply:
Forgot to switch the flag when I was testing. Thank u
```suggestion
      if (IS_TESTING !== 'true') {
```